docs_list.md

```diff
@@ -0,0 +1,143 @@
+# Docs list
+
+A working list of docs with links to the published documentation, the repo or repos, and the intended purpose of the technical content.
+
+## Cosmos Hub
+
+- [Cosmos Hub](https://hub.cosmos.network/main/hub-overview/overview.html)
```

Comment:
```suggestion
- [hub.cosmos.network](https://hub.cosmos.network/main/hub-overview/overview.html)
```
nice to be able to see also the domain since the title is part of the header above